src/pages/team.tsx

```diff
@@ -106,7 +108,9 @@ const Team: NextPage = () => {
         </p>
         <div className="grid grid-cols-8 md:grid-cols-16 lg:grid-cols-18 xl:grid-cols-20 mt-8 gap-2">
           {contributors
-            ?.filter((contributor) => contributor.id !== 1007849)
+            ?.filter(
+              (contributor) => HIDDEN_USERS.indexOf(contributor.id) === -1
```

Comment:
Can we not use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes?

Reply:
Didn't know that was a thing, nice